pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -401,7 +401,7 @@ func TestEnoughRequests(t *testing.T) {
 	for _, test := range enoughPodsTests {
 		t.Run(test.name, func(t *testing.T) {
 			node := v1.Node{Status: v1.NodeStatus{Capacity: makeResources(10, 20, 32, 5, 20, 5).Capacity, Allocatable: makeAllocatableResources(10, 20, 32, 5, 20, 5)}}
-			test.nodeInfo.SetNode(&node)
+			_ = test.nodeInfo.SetNode(&node)
```

Comment:
Is this necessary?

Reply:
No, the point here is that this func may return an error somethings, but we know it will not in this test or we can safely ignore it.

Reply:
I don't think it's a common pattern used in Go projects. Correct me if I'm wrong. Otherwise, let's keep the original logic.

Reply:
reverted

Reply:
Just FYI, get `Do not discard errors using _ variables.` when going through https://github.com/golang/go/wiki/CodeReviewComments#handle-errors.